nebula_messages/continental_msgs/msg/NebulaPackets.msg

```diff
@@ -0,0 +1,2 @@
+std_msgs/Header   header
+NebulaPacket[]   packets
```

Comment:
This seems to be a duplicate of `nebula_messages/nebula_msgs/msg/NebulaPackets.msg` and should be removed here.

Reply:
Thanks for the catch !

Reply:
@mojomex 
Addressed in 
b7f36e3